stock_view_adj_hls/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends field in stock_quant_view and following stock_move_line view.
```

Comment:
```suggestion
This module makes adjustments to some of the stock related views.

Some points to note:

* `lot_name` in stock move line operation tree is made invisible in view of using
  the OCA module stock_picking_auto_create_lot - the users will not be manually
  keying in the lot number during the receipt operation.
```

Please re-generate README.rst.

Reply:
I edited DESCRIPTION and re-generate README.rst